Gemfile

```diff
@@ -97,6 +97,7 @@ gem 'sassc',                    '~> 2.0'
 gem 'sass-rails',               '~> 5'
```

Comment:
I think this line got dup in the merge, I don't remember what will happen, maybe bundler will just ignore it